src/app/api/routing.py

```diff
@@ -98,7 +98,7 @@ async def heartbeat(device_table: Table, device: DeviceOut) -> HeartbeatOut:
     return device
 
 
-async def update_location(device_table: Table, payload: UpdatedLocation, device_id: int, user_id: int):
+async def update_location(device_table: Table, payload: DefaultLocation, device_id: int, user_id: int):
```

Comment:
Should be DefaultPostion from your modification in routes/devices.py

Reply:
good catch! Strange how it wasn't catch by unittests this time again :thinking:

Reply:
It's the same JSON schema so I would assume it shouldn't catch an exception unless we look for the exact values?

Reply:
Well DefaultLocation does not have `yaw` or `pitch` as fields, and since we pass the full payload in the unittest, I was expecting it to return a validator error. But I might be missing something here